metricbeat/module/system/process/process.go

```diff
@@ -128,10 +128,11 @@ func (m *MetricSet) Fetch(r mb.ReporterV2) error {
 	for _, proc := range procs {
 		rootFields := common.MapStr{
 			"process": common.MapStr{
-				"name": getAndRemove(proc, "name"),
-				"pid":  getAndRemove(proc, "pid"),
-				"ppid": getAndRemove(proc, "ppid"),
-				"pgid": getAndRemove(proc, "pgid"),
+				"name":         getAndRemove(proc, "name"),
+				"pid":          getAndRemove(proc, "pid"),
+				"ppid":         getAndRemove(proc, "ppid"),
+				"pgid":         getAndRemove(proc, "pgid"),
+				"command_line": getAndRemove(proc, "cmdline"),
```

Comment:
Do we really want to backport this, or will the `alas` just keep the old value?

Reply:
yep I just updated the PR to duplicate this field instead of replacing it using ECS field.